### ISSUE TITLE
Geoportal looks for index [metadata_v1] but harvester indexes to [metadata] index

### ISSUE THREAD

**Issue by MapZombie** [Esri/geoportal-server-harvester#18]
I now understand why I was having great difficulties at https://github.com/Esri/geoportal-server-catalog/issues/42

- I deleted my index.
- I had then upgraded to the latest Harvester.  
- Tried to harvest again but only received errors in Geoportal.
- 
- Upgraded to ES 5+ and Geoportal 2.5
- Got the same errors

Then brought back my old index through a backup and the previous Geoportal worked again.

Basically it seems the new Harvester saves to the ES [metadata] index instead of [metadata_v1]

This took an entire day to figure out as I couldn't see anything about this in the logs.

**Comment by MapZombie** [comment]
Where can this parameter be changed?

**Comment by pandzel** [comment]
Output broker for Geoportal Server has a field allowing to declare index name (by default is blank).

**Comment by MapZombie** [comment]
Where exactly is this field?
